### DISCUSSION TITLE
default code snippets for code exporter

### DISCUSSION THREAD

**Discussion by sahingoko** [graphql/graphiql#3401] [Q&A / Support]
Hi,
I am using the latest code exporter plugin inside a react app, hpwever could not find how I can use the default code snippets.
All the sample codes have the exampleSnippetOne and exampleSnippetTwo. I simply want to use whatever built-in snippet comes with the plugin itself. 
Here is what the readme describes;
_const snippets = [exampleSnippetOne, exampleSnippetTwo]
const exporter = codeExporterPlugin({
  snippets,
  codeMirrorTheme: 'graphiql',
});_

There is no way to leave it empty or set the defaults.

Thanks. 
Ozlem

**Comment by acao** [comment]
this example shows how to create your own snippets:

https://github.com/graphql/graphiql/blob/main/examples/graphiql-webpack/src/snippets.js

as you can see, you can generate snippets for any language and it will render them in the codemirror editor mode for that language

there are no defaults. to set them as empty, you can set an empty array for snippets.
